### PR TITLE
doc/cephadm: remove confusing fragment from cephadm adoption step 9

### PR DESCRIPTION
**Pull request by zdover23**
## Checklist
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by varshar16** [comment]
Please do `git rebase origin/master`.

**Comment by sebastian-philipp** [comment]
> What I don't get yet is why zdover23/master (locally known as origin/master) keeps getting polluted. 

never never never never ever commit anything to your master branch.

> I am grateful that I am working here with version control and not chainsaws, or I would be, as I have said elsewhere, a limbless torso with a head screaming about how confusing chainsaws are before it bleeds to death.

Yeah, turns out git has a steep learning curve (aka bad ux). But here we are. The world has decided that git is the way to go.

**Comment by zdover23** [comment]
> never never never never ever commit anything to your master branch.

I am truly sorry that I committed a .yml file to my master branch in March in an attempt to test a potential solution to the problem of our docs website having bad search. I shall never again do anything even remotely like that, no matter how desperately bad search is on future projects. But I'm already in this situation, and it's too late to prevent the polluting commits from having happened. The way that git is set up feels like the American justice system, where a single error at one time in the past results in damnation an institutionally-mandated suffering for the rest of one's life. All I want to do is to clean out my working directory so that the commit history of the master branch of zdover23:ceph cleanly tracks the commit history of the master branch of ceph:ceph. But, like the spectre of nationalism in the post-Napoleonic era, these bad commits that pollute my master branch rise again and again to compromise any progress I make. I am truly repentant. What prayer can I offer to git so that it will be convinced of my contrition and relieve me of the burden of these hardy, unwelcome commits?
 
> Yeah, turns out git has a steep learning curve (aka bad ux). But here we are. The world has decided that git is the way to go.

I am not opposed to the world using git. I am opposed only to this mystifying situation in which these bad commits follow me like the eye of God following Cain across the desert.

**Comment by sebastian-philipp** [comment]
> > never never never never ever commit anything to your master branch.
> 
> I am truly sorry that I committed a .yml file to my master branch in March in an attempt to test a potential solution to the problem of our docs website having bad search. I shall never again do anything even remotely like that, no matter how desperately bad search is on future projects. But I'm already in this situation, and it's too late to prevent the polluting commits from having happened. The way that git is set up feels like the American justice system, where a single error at one time in the past results in damnation and institutionally-mandated suffering for the rest of one's life. All I want to do is to clean out my working directory so that the commit history of the master branch of zdover23:ceph cleanly tracks the commit history of the master branch of ceph:ceph. But, like the spectre of nationalism in the post-Napoleonic era, these bad commits that pollute my master branch rise again and again to compromise any progress I make. I am truly repentant. What prayer can I offer to git so that it will be convinced of my contrition and relieve me of the burden of these hardy, unwelcome commits?

`git checkout master && git reset --hard upstream/master`

any you're done.
> 
> > Yeah, turns out git has a steep learning curve (aka bad ux). But here we are. The world has decided that git is the way to go.
> 
> I am not opposed to the world using git. I am opposed only to this mystifying situation in which these bad commits follow me like the eye of God following Cain across the desert.

**Comment by sebastian-philipp** [comment]
jenkins test dashboard backend

**Comment by sebastian-philipp** [comment]
ping @zdover23

File: doc/cephadm/adoption.rst [hunk 1]
```diff
@@ -111,7 +111,8 @@ Adoption process
 
 #. Redeploy MDS daemons by telling cephadm how many daemons to run for
    each file system.  You can list file systems by name with ``ceph fs
-   ls``.  For each file system::
+   ls``. 
+   Run the following command on the master nodes::
```
Comment (varshar16) [inline doc/cephadm/adoption.rst:115]: nit:
```suggestion
   ls``.  Run the following command on the master nodes::
```

**Comment by zdover23** [comment]
jenkins render docs

**Comment by zdover23** [comment]
jenkins test dashboard backend

**Pull request merged**